service/analysis.gradle

```diff
@@ -0,0 +1,30 @@
+jacocoTestReport {
+	reports {
+		html.required = true
+		xml.required = true
+	}
+
+	dependsOn test
+
+	// https://www.baeldung.com/jacoco-report-exclude#2-gradle-configuration
+	afterEvaluate {
+		classDirectories.setFrom(files(classDirectories.files.collect {
+			fileTree(dir: it, exclude: ['bio/terra/batchanalysis/generated/**'])
+		}))
+	}
```

Comment:
I wonder if this is necessary; the catalog and BPM project's don't have these lines. (Although I'm not sure if BPM's coverage reports are configured yet)